pihole_adlist_tool

```diff
@@ -333,6 +355,12 @@ if [ "$REGEX_MODE" -eq 1 ];
         echo -e "  [i]  REGEX_MODE: Disabled"
 fi        
 
+# verify if there is an update available
+remote_version=$(fetch_remote_version)
+if [ "$remote_version" != "$PIHOLE_ADLIST_TOOL_VERSION" ]; then
+	echo -e "  [i]  There is an update available: v$remote_version"
```

Comment:
```suggestion
if [ "$remote_version" != "$PIHOLE_ADLIST_TOOL_VERSION" ]; then
    echo
	echo -e "  [i]  There is an update available: $remote_version"

    else
    echo -e "  [i]  PIHOLE_ADLIST_TOOL_VERSION: $PIHOLE_ADLIST_TOOL_VERSION"
fi
```

Reply:
> echo -e "  [i]  PIHOLE_ADLIST_TOOL_VERSION: $PIHOLE_ADLIST_TOOL_VERSION"

Wouldnt printing the current version of the tool here be a bit too verbose (I dont know what to call it, tbh)
Cause with this change, you would start the program get the "Pihole Adlist Tool" message, and later on get the somewhat same message?

What do you think of changing the initial message to include the version number as well?

Reply:
> What do you think of changing the initial message to include the version number as well?

As part of the header? Sounds good :)

Reply:
I really complicated the wording there ahah. But yes, in the header :)